integration_tests.py

```diff
@@ -79,6 +79,23 @@ def test(self):
             self.assertEqual(query.status, 'SUCCESS')
             self.assertListOfDictsEqual(query.to_dict(), expected)
 
+class KillQueryTest(MyriaTestBase):
+    def test(self):
+        # This is a long-running program
+        program = """
+x = [0 as exp, 1 as val];
+do
+  x = [from x emit exp+1 as exp, 2*val as val];
+while [from x emit max(exp) < 1000];
```

Comment:
May as well make this limit humongous (say a billion) to avoid races, since we're going to kill it at the first opportunity.